chia/full_node/weight_proof.py

```diff
@@ -1245,7 +1245,7 @@ def validate_recent_blocks(
             if sub_slot.challenge_chain.new_difficulty is not None:
                 diff = sub_slot.challenge_chain.new_difficulty
 
-        if (challenge is not None) and (prev_challenge is not None) and transaction_blocks > 1:
+        if (challenge is not None) and (prev_challenge is not None) and transaction_blocks > 2:
```

Comment:
this change looks unrelated. Could you add a comment to the code explaining why this should be `> 2`?

Reply:
this is to make sure we have the required depth in the cash, before we where only enforcing one transaction block which is not enough now since that transaction block might be in the same signage point